my.css

```diff
@@ -163,3 +171,185 @@ body {
+    display: flex;
+    justify-content: center;
+    max-width: 1150px;
+    height: 750px;
+    z-index: 1;
+    margin: 0 auto;
+  }
+
+  .wraper::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 400px;
+    height: 800px;
+    background-color: black;
+    z-index: -1;
+  }
+
+  .burgermenu {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 72px;
+    height: 72px;
+    background: lightgray;
+  }
+
+  .newcollection {
+    display: flex;
+    flex-direction: column;
+    margin: 10px;
+    width: 60%;
+    height: 450px;
+  }
+
+  .newcollection__title {
+    width: 70%;
+    margin: 0;
+    font-size: 50px;
+    line-height: 50px;
+  }
+
+  .newcollection__text {
+    line-height: 20px;
+    padding-bottom: 20px;
+    font-size: 13px;
+  }
+
+  .collection__left {
+    width: 40%;
+    margin: 40px;
+    padding: 40px;
+    display: flex;
+  }
+
+  .collection__img {
+    width: 100%;
+    height: auto;
+  }
+
+  .collection__image-source {
+    align-items: flex-start;
+    height: 100%;
+    padding-left: 0;
+  }
+
+  .newcollection__footer {
+    padding: 10px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .newcollection__link {
+    font-size: 10px;
+    line-height: 5px;
+  }
+}
+
+/* разрешение 1130 */
+
+@media screen and (max-width: 768px) {
+  .header {
+    position: relative;
+    display: inline-block;
+    max-width: 100%;
+    height: 69px;
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  .wraper::before {
+    display: none;
+  }
+
+  .collection__maus {
+    display: none;
+  }
+
+  .collection__icon {
+    display: none;
+  }
+
+  .collection__left {
+    display: none;
+  }
+
+  .newcollection__text {
+    display: none;
+  }
+
+  .newcollection__title {
+    display: none;
+  }
+
+  .newcollection__footer {
+    display: none;
+  }
+
+  .logo__promo {
+    color: #000;
+    padding-bottom: 30px;
+  }
+
+  .ru__promo {
+    display: none;
+  }
+
+  .collection {
+    background-image: url(fon/adaptiv.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    width: 100%;
+    height: calc(100vh - 69px);
+    padding-top: 0;
+    max-height: 528px;
+  }
+
+  .burgermenu {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 70px;
+    height: 69px;
+    background: lightgray;
+  }
+
+  .collection-title__adaptiv {
+    width: 297px;
+    height: 103px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 50px;
+    line-height: 51px;
+  }
+
+  .collection__adaptiv {
+    position: absolute;
+    display: block;
+    display: flex;
+    justify-content: flex-end;
+    flex-flow: column;
+    align-self: flex-end;
+  }
+
+  .collection__vector {
+    display: flex;
+    justify-content: center;
+  } 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5608084)